65black/StackOverflow-lite
# Ft validate all routes 159896225

#### What does this PR do?
This PR adds validation to all POST routes in StackOverflow-lite's API.

#### Description of Task to be completed?
Any post request to add an answer or question to the database without the required details will be rejected.

#### How should this be manually tested?
This is the syntax of a question and all the required data needed: 
```
question = {
    authorId: 1,
    title: 'Question 4',
    content: 'This is a sample question.',
    date: '1-08-2018'
  };
```
To see the validation in action, make a POST request to api/v1/questions/ with the following data: 
```
question = {
    authorId: 1,
    content: 'This is a sample question.',
    date: '1-08-2018'
  };
```
Notice that the 'title' field has been omitted. This POST request will return an error.
#### What are the relevant pivotal tracker stories?
#159896225
